### ISSUE TITLE
Party frames improvments ?

### ISSUE THREAD

**Issue by sigzegv** [Ko0z/UnitFramesImproved_Vanilla#31]
Do you plan some improvements pour party frames ? I really would love to see party targets next to party frames as vanilla frames.

**Comment by Ko0z** [comment]
Sounds like a nice feature, I'll look into it!

**Comment by phendrome** [comment]
Not only party frames. Raid frames too. The original ones need work bad.

**Comment by Ko0z** [comment]
@phendrome Yes, I agree and am working on it. I have been a little side tracked becuase of a crazy idea on making a complete UI. Progress has been made and I have completely rewritten this Unitframes addon within that one. I'm soon ready to enhance the features of unitframes further... take a look here if you'd like a sneak peek [https://gitlab.com/Ko0zi/zui](url)

**Comment by phendrome** [comment]
That looks sick! I have a similar UI currently with a lot of _newer_ addons mixed with some older that are kinder on the memory management. 

There's a few things here that are not seen in the screenshot.

- Modified MSBT 
- MPOWA for tracking your own buffs to your liking
- enemyFrames for tracking enemy debuffs and casting
- NECB for tracking DR and CD's (in the middle between the frames)

You got Discord? Would love to talk to you more about this!

Some addons worth of mention that are not mentioned are ChatSuey, RogueSpam and SUCC-bag along with others.

![classic](https://user-images.githubusercontent.com/2565750/47460558-0e284880-d7df-11e8-8549-c11971e30ca8.png)

**Comment by QuadrozZz** [comment]
would be so good to see party frames some biggest (scaled 120%/130%) and health bar taller than default (like player/target frames), may be optional.

thanks for yours great work anyway! =)

**Comment by Ko0z** [comment]
@phendrome @QuadrozZz Thanks for your suggestions, I have implemeted some of your ideas and have come a long way now with zUI, it will very soon be available for testing. I’ll post here again when it’s out!
